frontend/tet/admin/src/components/header/Header.tsx

```diff
@@ -40,13 +40,18 @@ const Header: React.FC = () => {
   const isLoading = userQuery.isLoading;
   const isLoginPage = asPath?.startsWith('/login');
 
+  const titleClickHandler = () => {
+    void router.push('/');
+  };
+
   return (
     <BaseHeader
       title={t('common:appName')}
       skipToContentLabel={t('common:header.linkSkipToContent')}
       menuToggleAriaLabel={t('common:header.menuToggleAriaLabel')}
       languages={languageOptions}
       onLanguageChange={handleLanguageChange}
+      onTitleClick={titleClickHandler}
```

Comment:
You can actually use a hook from shared code here: `onTitleClick={useGoToFrontPage()}` and it should do the same.